docs/deployment/deployment-checklist.md

```diff
@@ -314,19 +314,22 @@ Semgrep requires the following permissions (scopes) to enable the authentication
 
 <IpAddresses />
 
-#### Features that make use of IP addresses
+You may instead use the [Semgrep Network Broker](/docs/semgrep-ci/network-broker), which provides an alternative to allowlisting these IP addresses and has separate requirements.
```

Comment:
```suggestion
Alternatively, you can use the [Semgrep Network Broker](/docs/semgrep-ci/network-broker) to facilitate secure access with Semgrep instead of allowlisting these IP addresses.
```